### DISCUSSION TITLE
How to exclude route from jwt authentication?

### DISCUSSION THREAD

**Discussion by shunnmugam** [Kong/kong#8375] [Help]
I've multiple services, JWT authentication is included in all services, need to exclude that on particular routes, like register a new user, and login routes, is it possible?

**Comment by shettyh** [comment]
You can create a separate service for those routes that doesnt require JWT auth. AFAIK there is no way to exclude a plugin at route level, can only include extra plugins at route level.

**Reply by shunnmugam** [reply]
But only for that route, do we need to create new service?, is it scalable?
